portfolio/pom.xml

```diff
@@ -50,4 +50,18 @@
       </plugin>
     </plugins>
   </build>
+</project> -->
+<build>
```

Comment:
Try to improve your indentation in same level tags.

<build> should have the same indentation as </build> and so on.